### ISSUE TITLE
Suggestion - RuneLite Chat History - Add option for history to expire after X time passes

### ISSUE THREAD

**Issue by juclor1993** [runelite/runelite#10528]
Title is self explanitory

I think the chat history plug-in is a great tool but i dont want to have to disable the plug-in, log out, then back in to clear a 12 hr old message from my friend that isnt online right now.

If there were an option to set a duration on how long messages stay on your screen, still retaining the option to keep the messages up indefinitely, would be a welcomed addition to the plug-in. 

Thanks for reading.
 Post if you agree/would like to add your perspective.

**Comment by dekvall** [comment]
I don't think that chat messages should expire, my stance is that it should be an active decision on the user's part when to remove chat history.

It looks like the problem you are having can be resolved by right-clicking either of these stones and then choosing  `Clear history`.

![chat-stones](https://user-images.githubusercontent.com/25151927/71670820-7e324700-2d71-11ea-9579-258bed2c49aa.png)
